package.json

```diff
@@ -16,19 +16,20 @@
     "qdequele <quentin@meilisearch.com>"
   ],
   "license": "MIT",
-  "main": "./dist/bundles/meilisearch.cjs",
-  "module": "./dist/bundles/meilisearch.mjs",
```

Comment:
EXPLAINER: `"exports"` is the replacement for these and we no longer officially support Node.js versions prior to 18, so we do not need the backwards compatibility.